libs/langchain-google-genai/src/chat_models.ts

```diff
@@ -186,7 +186,7 @@ export class ChatGoogleGenerativeAI
   private client: GenerativeModel;
 
   get _isMultimodalModel() {
-    return this.model.includes("vision");
+    return this.model.includes("vision") || this.model === 'gemini-1.5-pro-latest';
```

Comment:
I think locking it to gemini-1.5-pro-latest will cause frustration later.
Perhaps a leading string check on "gemini-1.5"?

Reply:
thats a good suggestion, we can do that